index.html

```diff
@@ -36,12 +36,14 @@
         <button type="submit" class="submit-btn">Sign up</button>
       </div>
 
-      <div class="form-section">
+      <div class="Logo">
```

Comment:
Change this back to `form-section` - the change here will break your page styling